pkg/types/scheduler.go

```diff
@@ -74,18 +74,19 @@ type ContainerRequest struct {
 	Cpu              int64           `json:"cpu"`
 	Memory           int64           `json:"memory"`
 	Gpu              string          `json:"gpu"`
+	GpuRequest       []string        `json:"gpu_request"`
 	GpuCount         uint32          `json:"gpu_count"`
 	SourceImage      *string         `json:"source_image"`
 	SourceImageCreds string          `json:"source_image_creds"`
 	ImageId          string          `json:"image_id"`
 	StubId           string          `json:"stub_id"`
 	WorkspaceId      string          `json:"workspace_id"`
 	Workspace        Workspace       `json:"workspace"`
-	Stub             StubWithRelated `json:"stub"`
 	Timestamp        time.Time       `json:"timestamp"`
 	Mounts           []Mount         `json:"mounts"`
 	RetryCount       int             `json:"retry_count"`
 	PoolSelector     string          `json:"pool_selector"`
+	Stub             StubWithRelated `json:"stub"`
```

Comment:
why did you move this? I thought it sort of made sense next to Workspace

Reply:
not sure why i moved it. moving back